src/app/app.ts

```diff
@@ -1,5 +1,11 @@
+import { GamePage, HomePage } from './components'
+
 export class App {
   start() {
     console.log('Starting application...')
+    const homepage = new HomePage()
+    const gamepage = new GamePage()
+    homepage.runPage()
+    gamepage.addQuitFunction()
```

Comment:
I am not sure if this is a good place to call this function. Maybe would it be better to call it in constructor of GamePage?

Reply:
You mean something like "new HomePage.runPage()" ?

Reply:
for now yes, We will find  a  better solution for it tomorrow